### ISSUE TITLE
A11y-Must: Remove smooth scroll

### ISSUE THREAD

**Issue by batemapf** [department-of-veterans-affairs/va.gov-team#9601]
>Please remove the smooth scroll from the automatic focus management, as it may trigger those with motion sensitivity.

From: https://github.com/department-of-veterans-affairs/va.gov-team/issues/9497#issuecomment-634177235

**Comment by batemapf** [comment]
Came from sample code on the design system / client application documentation. Maybe that needs to be updated too?

**Comment by batemapf** [comment]
@artsymartha68 will raise at the Design Council

**Comment by artsymartha68** [comment]
Advice from Robin Garrison via this Slack thread

https://dsva.slack.com/archives/C8E985R32/p1591299017158700?thread_ts=1591298527.158400&cid=C8E985R32

Tell your devs to add this setting when the component initializes (ComponentDidMount if you're using Classes)
window.VetsGov.scroll = {
  duration: 0,
  delay: 0,
  smooth: false
};

**Comment by mchelen-gov** [comment]
upstream feature request: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/312
